### ISSUE TITLE
Unclear error message (ChildMissingInParent)

### ISSUE THREAD

**Issue by ftomassetti** [LionWeb-io/lionweb-typescript#122]
_Originally created as https://github.com/LionWeb-io/lionweb-repository/issues/24_

When [sending a partition](https://github.com/LionWeb-io/lionweb-repository/files/14205752/sent.json) to the repository
I got a list of validation errors, the first being:

```
"ChildMissingInParent: Node with id \"Users_ftomassetti_repos_kolasu-java-langmodule_build_downloaded-examples_arthas_core_src_main_java_com_taobao_arthas_core_Arthas_java__root_declarations_0_members_0_exceptions_0\" has parent with id \"Users_ftomassetti_repos_kolasu-java-langmodule_build_downloaded-examples_arthas_core_src_main_java_com_taobao_arthas_core_Arthas_java__root_declarations_0_members_0\" but parent does not contains it as a child. at $.node[2] "
```

Followed by other similar errors.

This message confused me because actually the parent list the children correctly. The problem seems to be another: there are two parents containing the same child, for same reason. So probably this confuses the validation process and cause it produce this error message.

**Comment by joswarmer** [comment]
Fixed and released as ` @lionweb/validation@0.6.1-beta.0`

**Comment by joswarmer** [comment]
Fixed and released as  @lionweb/validation@0.6.1-beta.0
